src/util/async/AnyExecutionContext.hpp

```diff
@@ -211,14 +208,14 @@ class AnyExecutionContext {
 
         virtual impl::ErasedOperation
         execute(
-            std::function<impl::Any(AnyStopToken)>,
+            std::function<std::any(AnyStopToken)>,
```

Comment:
Not related to the PR but I'm curious why did you need to use any with type erased stop token?